src/lib/ydocker/pull_image_dialog.rb

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2017 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+# Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact Novell about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+
+require "docker"
+require "yast"
+
+module YDocker
+  class PullImageDialog
+    include Yast::UIShortcuts
+    include Yast::I18n
+    extend Yast::I18n
```

Comment:
why extend is needed? it should not be necessary as all translations are done in instance methods.

Reply:
> why extend is needed? it should not be necessary as all translations are done in instance methods.

Deleted.